ang/workflow/action-links/directives/workflow-list-edit-action.html

```diff
@@ -0,0 +1 @@
+<a>{{ ts('Edit') }}</a>
```

Comment:
@deb1990 are the duplicate and edit actions implemented? This PR is pointing to master. Is it something that can be released without causing confusion to users?

Reply:
Thats a good point, also it wont break things. So I will confirm with Fran/Sohaib 👍